src/Umbraco.Core/Runtime/IRuntimeMinifier.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Umbraco.Core.Assets;
+
+namespace Umbraco.Core.Runtime
```

Comment:
This shouldn't be in this namespace, although it's called "RuntimeMinifier" its not part of the actual Umbraco "Runtime". 

There's a few namespaces here that should just be one: Umbraco.Core.Assets is specifically to do with this so they should all be in the same namespace. This is also specifically for "Web" so I'd suggest it be part of an Umbraco.Web.RuntimeMinification namespace which sort of corresponds to the Umbraco.Web.Common.RuntimeMinification. But I'd go a step further, we have other web namespaces in Umbraco.Core that are new that should be consolidated:

* Cookie
* Request
* Search (SearchableTreeAttribute move to Trees)
* Session

I'll create a task for that but will move this and Assets into one namespace